webapp/components/admin_console/webhook_settings.jsx

```diff
@@ -107,7 +107,7 @@ export default class WebhookSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.integrationAdmin'
-                            defaultMessage='Enable Integrations for Admin Only: '
+                            defaultMessage='Restrict Integrations to Admins: '
```

Comment:
Should this be `Restrict creation of integrations to Team Admins and System Admins`?

Help text is vague, it is team admins AND system admins? 
![image](https://cloud.githubusercontent.com/assets/177788/16301901/1f846f06-3916-11e6-910f-6ceb1da25ee1.png)